docs/modules/operation/pages/deep-dive/service-assurance/scheduled-outages.adoc

```diff
@@ -0,0 +1,45 @@
+
+= Scheduled Outages
+
+Scheduled Outages allow for suppressing notifications and pausing Status Polling, Threshold Checking, and Data Collection in {page-component-title}.
+{page-component-title} supports many configurable options for when scheduled outages need to occur.
+
+Scheduled Outages is located under Admin -> Service Monitoring -> Configure Scheduled Outages
+
+== Components of a Scheduled Outage
+A Scheduled Outage definition requires the following:
+
+. Schedule Outage Name
+. Nodes to include in the scheduled outage
+. Interfaces on a node to include in the scheduled outage
+. Outage type Specific/Daily/Weekly/Monthly
+. Date/Time the outage occurs
+. Suppression of the following
+** All Notifications
+** Status Polling
+** Threshold Checking
+** Data Collection
```

Comment:
```suggestion
. Name.
. Nodes to include.
. Node interfaces to include.
. Outage type (daily, weekly, monthly, custom).
. Date and time for the outage.
. Suppression settings (all notifications, status polling, threshold checking, data collection).
```

Reply:
For the Outage type, it isn't custom, it is specific.
<img width="134" alt="Screenshot 2023-02-17 at 3 13 12 PM" src="https://user-images.githubusercontent.com/111781759/219814761-93cecca1-abf2-4069-b6c6-db1207cadc6d.png">